packages/storybook-ui-components/.storybook/preview.ts

```diff
@@ -1,5 +1,5 @@
 // .storybook/preview.ts
-import "@cockroachlabs/ui-components/dist/main.css"
+import "@cockroachlabs/ui-components/dist/cjs/index.css"
```

Comment:
Huh, I tried removing this import and it still failed to build for me. I guess it was just the type errors!